flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/ParquetColumnarRowInputFormatTest.java

```diff
@@ -85,45 +91,43 @@ public class ParquetColumnarRowInputFormatTest {
-                    primitive(PrimitiveTypeName.FLOAT, Repetition.OPTIONAL).named("f6"),
-                    primitive(PrimitiveTypeName.DOUBLE, Repetition.OPTIONAL).named("f7"),
-                    primitive(PrimitiveTypeName.INT96, Repetition.OPTIONAL).named("f8"),
-                    primitive(PrimitiveTypeName.INT32, Repetition.OPTIONAL)
-                            .precision(5)
-                            .as(OriginalType.DECIMAL)
-                            .named("f9"),
-                    primitive(PrimitiveTypeName.INT64, Repetition.OPTIONAL)
-                            .precision(15)
-                            .as(OriginalType.DECIMAL)
-                            .named("f10"),
-                    primitive(PrimitiveTypeName.BINARY, Repetition.OPTIONAL)
-                            .precision(20)
-                            .as(OriginalType.DECIMAL)
-                            .named("f11"),
-                    primitive(PrimitiveTypeName.FIXED_LEN_BYTE_ARRAY, Repetition.OPTIONAL)
-                            .length(16)
-                            .precision(5)
-                            .as(OriginalType.DECIMAL)
-                            .named("f12"),
-                    primitive(PrimitiveTypeName.FIXED_LEN_BYTE_ARRAY, Repetition.OPTIONAL)
-                            .length(16)
-                            .precision(15)
-                            .as(OriginalType.DECIMAL)
-                            .named("f13"),
-                    primitive(PrimitiveTypeName.FIXED_LEN_BYTE_ARRAY, Repetition.OPTIONAL)
-                            .length(16)
-                            .precision(20)
-                            .as(OriginalType.DECIMAL)
-                            .named("f14"));
+    private static final RowType ROW_TYPE =
+            RowType.of(
+                    new VarCharType(VarCharType.MAX_LENGTH),
+                    new BooleanType(),
+                    new TinyIntType(),
+                    new SmallIntType(),
+                    new IntType(),
+                    new BigIntType(),
+                    new FloatType(),
+                    new DoubleType(),
+                    new TimestampType(9),
+                    new DecimalType(5, 0),
+                    new DecimalType(15, 0),
+                    new DecimalType(20, 0),
+                    new DecimalType(5, 0),
+                    new DecimalType(15, 0),
+                    new DecimalType(20, 0),
+                    new ArrayType(new VarCharType(VarCharType.MAX_LENGTH)),
+                    new ArrayType(new BooleanType()),
+                    new ArrayType(new TinyIntType()),
+                    new ArrayType(new SmallIntType()),
+                    new ArrayType(new IntType()),
+                    new ArrayType(new BigIntType()),
+                    new ArrayType(new FloatType()),
+                    new ArrayType(new DoubleType()),
```

Comment:
Add a test like `new ArrayType(new ArrayType(new DoubleType()))`?

Reply:
Parquet writer doesn't support writer `new ArrayType(new ArrayType(new DoubleType()))` type now, so this type of test is not good to add, please refer to `ArrayWriter`.